en/core-libraries/collections.rst

```diff
@@ -1178,6 +1178,35 @@ places at the same time. In order to clone a collection out of another use the
         }
     }
 
+Creating your own custom Collections
+====================================
+
+It is possible to create your own collection classes in your application to store, manage and manipulate groups of items. Simply create your collection class and extend the core collection class instance.
+
+    namespace App\Lib\Collections;
+
+    use Cake\Collection\Collection;
+
+    class ExamplesCollection extends Collection
+    {
+
+    }
+
+You can now use your collection class as you would a regular collection, and access all the power of the collection methods.
+
+    $examplesCollection = new ExamplesCollection(['first' => 'Custom class example', 'second' => 'Extend core class']);
+
+If you are storing object instances in a custom collection it is recommended to include a key when constructing the collection.
```

Comment:
comma missing in if sentence.

Reply:
Can you tell me where as it reads okay to me.